functions.gallery.php

```diff
@@ -7,6 +7,15 @@ class Jetpack_Gallery_Settings {
 	function __construct() {
 		add_action( 'admin_init', array( $this, 'admin_init' ) );
 	}
+	
+	/**
```

Comment:
You'll need to place the inline doc right above the line including the filter. There should be nothing between the filter and its comment.